guava/src/com/google/common/collect/ForwardingMap.java

```diff
@@ -97,21 +97,18 @@ public void clear() {
 
   @Pure
   @Override
-  @SuppressWarnings("nullness") // Suppressed due to annotations on containsKey in Java.Map
```

Comment:
Removing these `@SuppressWarnings` may be the right thing to do, but it isn't logically part of the merge.